<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F3E9E2;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px ridge #673C4F;

background-color: #F9F4F1;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 40pt &quot;Skia&quot;;
color: #673C4F;
</string>
    </property>
    <property name="text">
     <string>Pantry Pal</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>571</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px ridge #673C4F;
background-color: #F9F4F1;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Skia&quot;;
color: #673C4F;
background: rgb(204, 213, 174);
border: black</string>
    </property>
    <property name="text">
     <string>Find Recipe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Skia&quot;;
color: #673C4F;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListWidget" name="pantryListWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>271</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #673C4F;
font: 20pt &quot;Skia&quot;;
border: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 17pt &quot;Skia&quot;;
color: #673C4F;
</string>
    </property>
    <property name="text">
     <string>Pantry:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 15pt &quot;Skia&quot;;
color: #673C4F;
</string>
    </property>
    <property name="text">
     <string>Made by Amelia Reeves and Connor Curcio</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: 9px dotted rgb(204, 213, 174);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: 9px dotted rgb(204, 213, 174);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="recipeNameLineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="text">
     <string>Name of Recipe</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ingredientsTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Ingredients:
1/2 cup example
1 Tbs example
example
example</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="directionsTextEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>261</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Directions:
1. Step 1
2. Step 2
3. Step 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="servingsLineEdit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="text">
     <string>Servings: 10</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt &quot;Skia&quot;;
color: #673C4F;
background: rgb(204, 213, 174);
border: black</string>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dfsLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="text">
     <string>DFS: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="bfsLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Skia&quot;;
color: #673C4F;
border: transparent;</string>
    </property>
    <property name="text">
     <string>BFS:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bfsCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dfsCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
